service/src/main/kotlin/no/nav/su/se/bakover/service/sak/SakServiceImpl.kt

```diff
@@ -26,6 +27,10 @@ internal class SakServiceImpl(
         return sakRepo.hentSak(fnr)?.right() ?: FantIkkeSak.left()
     }
 
+    override fun hentSak(saksnummer: Saksnummer): Either<FantIkkeSak, Sak> {
+        return sakRepo.hentSak(saksnummer)?.right() ?: FantIkkeSak.left()
```

Comment:
Kun til info for de arrow frelste: Tror man og kan gjøre noe ala. `return sakRepo.hentSak(saksnummer).rightIfNotNull{FantIkkeSak}`

Reply:
Haha niceeeeee, men de `hentSak(..)` for `sakId` og `fnr` er skrivna på det sättet där uppe ☺️ . Tänker att det kan vara fint att ha det likt?

Reply:
Jepp, bedre å ha det likt.